service/controller/internal/vmsku/vmsku.go

```diff
@@ -0,0 +1,129 @@
+package vmsku
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/azure-operator/v4/client"
+)
+
+const (
+	// CapabilitySupported is the value returned by this API from Azure when the capability is supported
+	CapabilitySupported = "True"
+
+	CapabilityAcceleratedNetworking = "AcceleratedNetworkingEnabled"
+)
+
+type Config struct {
+	ClientFactory *client.Factory
+	Location      string
+	Logger        micrologger.Logger
+}
+
+type VMSKUs struct {
+	clientFactory *client.Factory
+	location      string
+	skus          map[string]*compute.ResourceSku
+	logger        micrologger.Logger
+	mux           sync.Mutex
```

Comment:
Minor nitpick: _mux_ as term is often used in context of HTTP handlers path routing e.g. [stdlib](https://golang.org/pkg/net/http/#ServeMux). Perhaps `initMutex` ?

Reply:
makes a lot of sense. Done, thanks